libsolidity/analysis/ContractLevelChecker.cpp

```diff
@@ -220,7 +220,9 @@ void ContractLevelChecker::checkAbstractDefinitions(ContractDefinition const& _c
 		for (auto declaration: _contract.annotation().unimplementedDeclarations)
 			ssl.append("Missing implementation: ", declaration->location());
 		m_errorReporter.typeError(
-			3656_error,_contract.location(), ssl,
+			3656_error,
+			_contract.location(),
+			ssl,
 			"Contract \"" + _contract.annotation().canonicalName
 			+ "\" should be marked as abstract.");
```

Comment:
Style: the last bracket should be on the next line? The operator `+` should be on the line above.